src/cufinufft.cu

```diff
@@ -1,17 +1,74 @@
 #include <iostream>
 #include <iomanip>
+#include <vector>
+#include <complex>
+#include <cstddef>
 #include <math.h>
 #include <helper_cuda.h>
 #include <complex>
 #include <cufft.h>
+// #include <thrust/zip_iterator.h>
+#include <thrust/tuple.h>
```

Comment:
Is thrust a new dependency? or is this just some headers that can be self-contained in the project?